apps/nerves_hub_www/lib/nerves_hub_www_web/templates/device/console.html.eex

```diff
@@ -1,7 +1,17 @@
+<div class="flex-row justify-content-between align-items-center">
+  <%= link "Back to device", to: Routes.device_path(@conn, :show, @org.name, @product.name, @device.identifier), class: "back-link" %>
+  <button class="btn btn-outline-light" id="restart_button" <%= if @console_available != true do "disabled" end%>>
```

Comment:
This can just be if `!@console_available` unless we're worried about that var being something besides a boolean.